pyomo/contrib/mindtpy/util.py

```diff
@@ -499,6 +502,16 @@ def set_solver_options(opt, solve_data, config, solver_type, regularization=Fals
             opt._solver_model.parameters.mip.strategy.presolvenode.set(3)
             if config.add_regularization in {'hess_lag', 'hess_only_lag'}:
                 opt._solver_model.parameters.optimalitytarget.set(3)
+    elif solver_name == 'appsi_cplex':
+        opt.options['timelimit'] = remaining
+        opt.options['mip_tolerances_mipgap'] = config.mip_solver_mipgap
+        # opt.options['load_solution'] = False
+        if regularization is True:
+            if config.solution_limit is not None:
+                opt.options['mip_limits_solutions'] = config.solution_limit
+            opt.options['mip_strategy_presolvenode'] = 3
+            if config.add_regularization in {'hess_lag', 'hess_only_lag'}:
+                opt.options['optimalitytarget'] = 3
     elif solver_name == 'glpk':
         opt.options['tmlim'] = remaining
         # TODO: mipgap does not work for glpk yet
```

Comment:
Maybe we should create an issue mentioning this, as it is an important feature of a MIP solver and many people use GLPK with Pyomo (at least as an starting solver)

Reply:
This comment is outdated. MIPGAP is already supported by GLPK with Pyomo. Fixed.